pkgs/applications/science/astronomy/kstars/default.nix

```diff
@@ -37,13 +37,11 @@
 
 stdenv.mkDerivation (finalAttrs: {
   pname = "kstars";
-  version = "3.6.9";
+  version = "3.7.0";
 
-  src = fetchFromGitHub {
-    owner = "KDE";
-    repo = "kstars";
-    rev = "stable-${finalAttrs.version}";
-    hash = "sha256-28RRW+ncMiQcBb/lybEKTeV08ZkF3IqLkeTHNW5nhls=";
+  src = fetchurl {
+    url = "mirror://kde/stable/kstars/kstars-${finalAttrs.version}.tar.xz";
+    sha256 = "sha256-yvN1k0LqUi5Odb34Nk8UP5qoIbFUcvUiyESpoMKmuqg=";
```

Comment:
Probably except the `hash -> sha256` change.

Reply:
fixed